runtime/.gitignore

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
will this be required once your PR lands?

Reply:
yea, it still puts it in the target directory unless `$OUT_DIR` is specified. If you just run `cargo test` inside /runtime then it will be there in runtime/target/..

Reply:
That's if a test panics for instance and the drop is not run.

Reply:
looks like OUT_DIR isn't set unless we have a build script in our project...  I think we might want to toss all refs to OUT_DIR and use CARGO_MANIFEST_PATH + "target/tmp"

Reply:
https://doc.rust-lang.org/cargo/reference/environment-variables.html

Reply:
Well maybe if OUT_DIR isn't set, then you fall-back to CARGO_MANIFEST_PATH/target/tmp/. Something may want to control OUT_DIR, and you can't do that if you are always using cargo manifest path. But I think just by giving a relative path seems like you get the CARGO_MANIFEST_PATH root by default. It's more like if we want to rely on that behavior I think.